### PR TITLE
Upgraded the dependency vimeo/vimeo-api to version 3.0.2

### PR DESCRIPTION
**Pull request by SafwanAd-Dardak**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-    "name": "vimeo/laravel",
+    "name": "safwan-ad-dardak/laravel",
```
Comment (muglug) [inline composer.json]: Is this change deliberate?

Comment (SafwanAd-Dardak) [reply composer.json]: nope that was a mistake

**Comment by SafwanAd-Dardak** [comment]
There you go. Sorry about that.

**Comment by muglug** [comment]
Mind merging/rebasing latest master?

**Comment by SafwanAd-Dardak** [comment]
It should be up-to-date now.

**Comment by muglug** [comment]
Thanks!
